### ISSUE TITLE
Change the title of the save dialog

### ISSUE THREAD

**Issue by Choppel** [OpenSourcePKG/findnow_tbplugin#8]
The dialog that lets you choose the destination for the saved file has no title. Is this adjustable?

**Comment by stefanwerfling** [comment]
There is a possibility:
`filepicker.init(window, dialogTitle, Ci.nsIFilePicker.modeSave);`
Which title should be set?

**Comment by Choppel** [comment]
"Speichern unter" in german, "Save to" in english.
Can the button label also be adjusted?

**Comment by stefanwerfling** [comment]
Yes we can:
`filepicker.okButtonLabel = 'Choppel';`
also "Save to"?

**Comment by Choppel** [comment]
Yes please. Although I am flattered that you would name the button Choppel we should use something more descriptive :)
"Speichern" in german, "Save" in english.
